docs-source/content/userguide/managing-domains/model-in-image/debugging.md

```diff
@@ -68,13 +73,17 @@ For example, assuming your domain UID is `sample-domain1` and your domain namesp
         1. WLSDPLY-05007: Model file /u01/wdt/models/model1.yaml,/weblogic-operator/wdt-config-map/..2020_03_19_15_43_05.993607882/datasource.yaml contains an unrecognized section: TYPOresources. The recognized sections are domainInfo, topology, resources, appDeployments, kubernetes
   ```
 
+{{% notice tip %}}
+When a model file error references a model file in your model config map, then you can correct the error by redeploying the configmap with a corrected model file and then initiating a domain restart or roll. Similarly, if a model file error references a model file in your model image, then you can correct the error by deploying a corrected image, modifying your domain resource to reference the new image, and then initiating a domain restart or roll.
```

Comment:
config map -> ConfigMap (Because config map is not an English word (configuration map is), we decided to use the Kubernetes camel case in every instance.)